pyproject.toml

```
@@ -21,7 +21,12 @@ skip = "migrations"
 name = "redash"
 version = "23.09.0-dev"
 description = "Make Your Company Data Driven. Connect to any data source, easily visualize, dashboard and share your data."
-authors = ["Your Name <you@example.com>"]
+authors = ["Arik Fraimovich <email>"]
```

```suggestion
authors = ["Arik Fraimovich <arik@redash.io>"]
```

(I should probably start monitoring this email address again 🫣)